lib/kb_quast/kb_quastImpl.py

```diff
@@ -274,9 +291,11 @@ def run_QUAST(self, ctx, params):
                 filepaths.append(p)
                 labels.append(l)
 
+        skip_glimmer = self.check_large_input(filepaths)
+
```

Comment:
Should log that you're skipping glimmer either here or in the exec or check_large methods